rubicon_ml/workflow/prefect/__init__.py

```diff
@@ -8,6 +11,13 @@ def _check_for_prefect_extras():
         raise ImportError(message)
 
 
+warnings.warn(
+    "The `rubicon_ml.workflow.prefect` module is deprecated and will be removed in an upcoming release."
```

Comment:
do we need it twice

Reply:
yeah, I wanna make sure its raised if the workflow module is imported at any level since we're deprecating all of it. removing the additional warnings makes some of the new tests fail

Reply:
oh wow, I thought having it in the init would cover all the cases . Good to know